docs/debugging-source.md

```diff
@@ -22,11 +22,11 @@ with a specific type, and cache it in your local symbols cache.
 
 ![F11 step into symbol](https://cloud.githubusercontent.com/assets/359239/5389259/74600502-8149-11e4-94f7-10dc79a0573f.gif)
 
-You can then set subsequent breakpoints inside the source code, to return to 
+You can then set subsequent breakpoints inside the source code, to return to
 in the debugging session.
 
 ### Acknowledgements
 
-Thanks to [Cameron Taggart](http://blog.ctaggart.com/) for building the
-[SourceLink](https://github.com/ctaggart/SourceLink) framework which we use
-to support this behaviour.
\ No newline at end of file
+Thanks to previous of [Cameron Taggart](http://blog.ctaggart.com/) and the newer implementation of
+[SourceLink](https://github.com/dotnet/sourcelink) which we use
+to support this behaviour.
```

Comment:
I had a couple small grammatical suggestions, and I also don't think we should link to an HTTP-only personal site. Thoughts? 
```suggestion
Thanks to Cameron Taggart for the initial implementation and the .NET foundation for ongoing support of [SourceLink](https://github.com/dotnet/sourcelink) which we use to provide source debugging.
```

Another option might be to link to his GitHub page as the current Source Link [repo does](https://github.com/dotnet/sourcelink/blob/main/README.md?plain=1#L9).